client/src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { Menu, Button, Icon } from 'semantic-ui-react';
+
+export default class Navbar extends Component {
+  state = {
+    companyName: 'Tripen',
+  };
+
+  render() {
```

Comment:
Why stateful components?

Reply:
Was experimenting with boilerplate code. Will amend this right away.